src/index.js

```diff
@@ -170,17 +193,22 @@ async function createSwiftEnvironment(templateName, templateFolder) {
     "ios-swift"
   );
 
+  if (nativePath !== "ios") {
```

Comment:
`nativePath` will always be the `writeDirPath` so I think you can get rid of this if block.